chore: lint code and text in integration tutorials (#1167)
wandb/docs: content/tutorials/integration-tutorials/keras.md

---

{{< cta-button colabLink="https://colab.research.google.com/github/wandb/examples/blob/master/colabs/keras/Use_WandbMetricLogger_in_your_Keras_workflow.ipynb" >}}
Use Weights & Biases for machine learning experiment tracking, dataset
versioning, and project collaboration.

{{< img src="/images/tutorials/huggingface-why.png" alt="" >}}

This notebook introduces the `WandbMetricsLogger` callback. Use this callback
for [Experiment Tracking]({{< relref "/guides/models/track" >}}). It will log
your training and validation metrics along with system metrics to Weights and
Biases.

## Setup and Installation

> First, let us install the latest version of Weights and Biases. We will then

Reviewer: ```suggestion
In the notebook, install Weights and Biases and sign in.
```
It's better to avoid "let us" and "we" in favor of the simple imperative. I believe both of these code blocks should be tagged up as notebooks.

from wandb.integration.keras import WandbMetricsLogger
```

> If this is your first time using W&B or you are not logged in, the link that
> appears after running `wandb.login()` will take you to sign-up/login page.
> Signing up for a [free account](https://wandb.ai/signup) is as easy as a few
> clicks.

Reviewer: ```suggestion
If your browser is not logged in to W&B, the link that
appears after running `wandb.login()` will take you to the login page. If you don't yet have an account, [sign up for free](https://wandb.ai/signup), then log in.
```

```python
wandb.login()
```

## Hyperparameters

> Use of proper config system is a recommended best practice for reproducible
> machine learning. We can track the hyperparameters for every experiment using
> W&B. In this colab we will be using simple Python `dict` as our config system.

Reviewer: ```suggestion
To reproduce a machine learning experiment, it's important to save the hyperparameters that the experiment uses. This example shows how to track an experiment's hyperparameters in a Python `dict`.
```
This sentence is very hard for me to understand (not due to your changes). Here is my attempt to help, which may in fact be incorrect.
